src/components/modules/fluid-card/fuild-card.styles.ts

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import type { StyledFluidCardProps } from './fluid-card.types';
+
+import { css, styled } from '@/styles';
+
+export const StyledFluidCardWrapper = styled.div<StyledFluidCardProps>`
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  border-radius: 16px;
+  padding: 24px;
+  box-shadow:
+    0 0 0 1px ${({ theme }) => theme.colors.button.border},
+    0px 2px 24px 0px ${({ theme }) => theme.colors.shadowDefault};
+
+  ${({ theme, $variant }) => {
+    switch ($variant) {
+      case 'default':
+        return css`
+          background: ${theme.colors.darkGrey};
+          gap: 32px;
+        `;
+      case 'dark':
+        return css`
+          background: ${theme.colors.inkBlack};
+        `;
+      case 'highlighted':
+        return css`
+          background-image: url('/dots_dark.svg'), ${theme.colors.blueGradient};
+          background-size: 120% 150%;
+          background-position: bottom 0% center;
+          box-shadow:
+            0px 2.4px 1.2px 0px rgba(201, 212, 251, 0.1) inset,
+            0px -1.2px 1.2px 0px rgba(16, 46, 151, 0.1) inset,
+            0px 2.4px 1.2px 0px rgba(201, 212, 251, 0.1) inset,
+            0px -1.2px 1.2px 0px rgba(16, 46, 151, 0.1) inset;
+        `;
+    }
+  }}
+`;
+
+export const StyledFluidCardContentWrapper = styled.div<StyledFluidCardProps>`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ $variant }) => ($variant === 'highlighted' ? '16px' : '20px')};
+`;
+
+export const StyledFluidCardText = styled.div<StyledFluidCardProps>`
+  & * {
+    font-size: ${({ theme }) => theme.fontSizes.text.m};
+    color: ${({ theme, $variant }) =>
+      $variant === 'highlighted' ? theme.colors.text.lightGrey : theme.colors.text.darkGrey};
```

Comment:
Let's keep the white color in the highlighted version in both themes, please. It doesn't look good with the dark text

![Screenshot 2024-02-28 at 16 13 44](https://github.com/bejamas/httptoolkit-website/assets/22168189/d787b79f-79d3-4a90-a918-2a1619951674)